docs/release-notes/snapcraft-8-13.rst

```diff
@@ -46,3 +59,116 @@ the following commands in your local repository to sync the submodule change:
     git submodule sync
     git submodule update --init --recursive
     git clean -ffd
+
+
+New .NET extensions
+~~~~~~~~~~~~~~~~~~~
+
+The experimental ``dotnet8``, ``dotnet9``, and ``dotnet10``
+:ref:`reference-dotnet-extensions` have been added to Snapcraft for ``core24`` snaps.
+
+
+Minor features
+--------------
+
+Old command aliases
+~~~~~~~~~~~~~~~~~~~
+
+This release adds deprecation warnings to old command aliases, and creates new names for
+all of them. Any command that began with "``list-``" was affected.
```

Comment:
```suggestion
Renamed commands
~~~~~~~~~~~~~~~~

In accordance with our latest design standards, all commands starting with
``list-`` have been renamed. The word "list" will no longer be used. The old
command names will continue working until Snapcraft 9.

The affected commands are:
```

I believe this is accurate, because we didn't switch everything to `list-` originally, right?

Reply:
We can cut the sentence about the next Snapcraft.